srcts/src/shiny/bind.ts

```diff
@@ -27,7 +28,7 @@ function valueChangeCallback(
   inputs: InputValidateDecorator,
   binding: InputBinding,
   el: HTMLElement,
-  allowDeferred: boolean
+  priority: InputPolicyOpts["priority"]
```

Comment:
If I'm reading the source correctly, `InputPolicyOpts["priority"]` is the same as `EventPriority`, right? If so, I think the second option is clearer because it doesn't require redirecting through `InputPolicyOpts`